src/Microsoft.ML.PCA/PcaTrainer.cs

```diff
@@ -33,12 +33,31 @@ namespace Microsoft.ML.Trainers
     // REVIEW: make RFF transformer an option here.
 
     /// <summary>
-    /// This trainer trains an approximate PCA using Randomized SVD algorithm
-    /// Reference: https://web.stanford.edu/group/mmds/slides2010/Martinsson.pdf
+    /// The <see cref="IEstimator{TTransformer}"/> for training an approximate PCA using Randomized SVD algorithm.
     /// </summary>
     /// <remarks>
-    /// This PCA can be made into Kernel PCA by using Random Fourier Features transform
+    /// <format type="text/markdown"><![CDATA[
+    /// To create this trainer, use [RandomizedPca](xref:Microsoft.ML.PcaCatalog.RandomizedPca(Microsoft.ML.AnomalyDetectionCatalog.AnomalyDetectionTrainers,System.String,System.String,System.Int32,System.Int32,System.Boolean,System.Nullable{System.Int32}))
+    /// or [RandomizedPca(Options)](xref:Microsoft.ML.PcaCatalog.RandomizedPca(Microsoft.ML.AnomalyDetectionCatalog.AnomalyDetectionTrainers,Microsoft.ML.Trainers.RandomizedPcaTrainer.Options)).
+    ///
+    /// [!include[io](~/../docs/samples/docs/api-reference/io-columns-regression.md)]
```

Comment:
>regression [](start = 70, length = 10)

RandomizedPCA is not the same as regression, is it?

Reply:
I didn't.

Reply:
I did ...